datafusion/physical-optimizer/src/limit_pushdown.rs

```diff
@@ -247,7 +246,15 @@ pub fn pushdown_limit_helper(
             }
         } else {
             // Add fetch or a `LimitExec`:
-            global_state.satisfied = true;
+            // If the plan's children have limit, we shouldn't change the global state to true,
```

Comment:
If the children's limit is `>=` the globe limit, can we push down the limit?

Reply:
Good question @xudong963 , added the slt testing for  children's limit is >= the globe limit, the limit should also support push down consistent with current behaviour, thanks!